src/ast/ast_validations.c

```diff
@@ -1592,29 +1592,33 @@ bool AST_ContainsErrors(const cypher_parse_result_t *result) {
 	return cypher_parse_result_nerrors(result) > 0;
 }
 
-static AST_Validation _AST_Validate_ParseResultRoot(const cypher_parse_result_t *result) {
+static AST_Validation _AST_Validate_ParseResultRoot(const cypher_parse_result_t *result,
```

Comment:
Please add comment describing this function and the issue it solves (multiple root element)